mail_optional_follower_notification/wizard/mail_compose_message.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class MailComposeMessage(models.TransientModel):
+    _inherit = "mail.compose.message"
+
+    notify_followers = fields.Boolean(default=True)
+
+    def send_mail(self, auto_commit=False):
+        ctx = self.env.context.copy()
+        for wizard in self:
+            ctx["notify_followers"] = wizard.notify_followers
+            wizard = wizard.with_context(ctx)
+            super(MailComposeMessage, wizard).send_mail(auto_commit=auto_commit)
```

Comment:
each wizard could have potentially it's own ctx, I'd do:
```suggestion
        for wizard in self:
            wizard = wizard.with_context(notify_followers=wizard.notify_followers)
            super(MailComposeMessage, wizard).send_mail(auto_commit=auto_commit)
```

Reply:
@simahawk I admit that the syntax is more elegant (a remaining odoo v8 syntax inheritance). But AFAIK, the context is the same for all the records into a recordset since the context is provided by the environment.